test/firebase_fcm_test.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:dartbase/dartbase.dart';
+import 'package:dartbase/fcm/message.dart';
+import 'package:test/test.dart';
+
+import 'test_config.dart';
+
+
+
+Future main() async {
+  Firebase firebase;
+  FCM fcm;
+  final nameRegExp = RegExp(r'projects/.*/messages/[0-9]+');
+
+  setUpAll(() async {
+    firebase = await Firebase.initialize(projectId, await ServiceAccount.fromFile(serviceAccountPath));
+
+    await FirebaseAuth.initialize();
```

Comment:
You're depending on the global FirebaseAuth instance here but are depending on the local instance of Firebase. I would instead do either global for both, or local for both. Consistency makes everything much more readable :)

Reply:
Turns out I don't need this so removing.